qe-tests/cypress/integration/tests/vmware/config_separate_mapping.ts

```diff
@@ -58,19 +65,32 @@ export const storageMapping: MappingData = {
   mappingPeer: storageMappingPeer,
 };
 
+export const preHookData: HookData = {
+  ansiblePlaybook: preAnsiblePlaybook,
+  //image: preImage,
+};
+
+/*export const postHookData: HookData = {
+  ansiblePlaybook: postAnsiblePlaybook,
+  //image: postImage,
+};*/
+
 export const planData: PlanData = {
   name: 'testplan-separate-mapping-cold',
   sProvider: providerData.name,
   tProvider: 'host',
   namespace: 'default',
-  sourceClusterName: 'Cluster',
-  vmwareSourceVmList: ['v2v-rhel7-igor'],
+  //sourceClusterName: 'Cluster',
```

Comment:
Please clean all comments of not-required code

Reply:
Thanks for reviewing, I'll will clean up the not-required code.